exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/ServiceRuntimeIntegrationTest.java

```diff
@@ -96,8 +96,6 @@ void startsServerOnInitialization() {
 
   @Test
   void deployCorrectArtifact() throws Exception {
-    serviceRuntime.initialize(mock(Node.class));
```

Comment:
(Here and elsewhere) Why is this removed? No operation shall occur before SR is initialized.

Reply:
ok, I will add a null check to the decorator then

Reply:
Not sure I understand — the SR expects that it is first initialized with a non-null Node before any other operations are performed.